house/views.py

```diff
@@ -11,7 +17,38 @@ def global_page(request):
 
 
 def house_login(request):
-    raise NotImplementedError
+
+    # if this is a POST request we need to process the form data
+    if request.method == 'POST':
+
+        # create a form instance and populate it with data from the request:
+        form = HouseIdForm(request.POST)
+
+        # check whether it's valid:
+        if form.is_valid():
+
+            house_id = form.data["house_id"]
+
+        # if User.objects.filter(email=cleaned_info['username']).exists():
+        #    return HttpResponseRedirect(reverse(LOGIN_PAGE_ROUTE))
+        if House.objects.filter(house_id=house_id).exists():
+
+            return HttpResponseRedirect('/../{house_id}')
+        # ////// OR //////
+        # if House.objects.get(house_id=house_id):
+        #    return HttpResponseRedirect('/../{house_id}')
+
+        else:
+            # Create a new House with the given house_id:
+
+            House.objects.create(house_id=house_id)
```

Comment:
You should create the object using the function and can fill up the data needed from
form.data[]
 You could alsoo, I think there is a way to make the form create the house with its magic we need to check that Django sorcery.